mono/metadata/icall.c

```diff
@@ -2163,25 +2163,39 @@ ves_icall_MonoField_SetValueInternal (MonoReflectionFieldHandle field, MonoObjec
 		mono_gchandle_free (value_gchandle);
 }
 
+ICALL_EXPORT MonoObject*
+mono_TypedReference_ToObject (MonoTypedRef* tref);
+
 ICALL_EXPORT void
 ves_icall_System_RuntimeFieldHandle_SetValueDirect (MonoReflectionField *field, MonoReflectionType *field_type, MonoTypedRef *obj, MonoObject *value, MonoReflectionType *context_type)
 {
+	HANDLE_FUNCTION_ENTER ();
+
+	ERROR_DECL (error);
+
 	MonoClassField *f;
+	MonoObjectHandle objHandle, valueHandle;
```

Comment:
Suggest move these down to the initialization instead of leaving them uninitialized and wasting vertical space.